### ISSUE TITLE
Add missing Tag attributes for Angular

### ISSUE THREAD

**Issue by Akshat55** [IBM/carbon-ui-builder#157]
Add missing attributes added in CCA PR [#2279](https://github.com/IBM/carbon-components-angular/pull/2279)

Referencing #134, to mark Tag as complete

**Comment by miqh** [comment]
Hello @Akshat55,

I had some curiosities (some of which aren't strictly about this particular issue) I was hoping you could clarify.

Between the Angular and React implementations of Carbon Design System, I noticed not all components have feature parity. The `Accordion` has one obvious example. The React version supports `isFlush`, which applies some stylistic tweaks. The Angular version doesn't have such an input at all.

Another more subtle example is `Button`. The Angular version supports `assistiveTextPlacement`, yet the React version (presumably) supports this as `tooltipPosition` (or it may be something else entirely). In such cases, are you expecting this project to churn out the minimum common subset of supported attributes and ignore those which aren't universal?

Specifically about `a-tag.tsx`, why does the Angular code export choose to include code which covers the `close` event emitter, yet the React version doesn't include any boilerplate for the `onClose` handler? (I assume this might just be for convenience.)

I can try make some changes if I understand the direction of this project a bit more.

**Comment by Akshat55** [comment]
Hi @miqh,

`isFlush` property was added in Carbon v11, currently Angular implementation (master branch) is using Carbon v10. Same applies to the button and tooltip positions. As for this reason, this project only supports Carbon v10 currently. You can find the storybook for Carbon v10 for react [here](https://v7-react.carbondesignsystem.com/) & the implementation [here](https://github.com/carbon-design-system/carbon/tree/v10/packages/react).

So for this issue, you'd have to update the angular code export in [a-tag.tsx](https://github.com/IBM/carbon-ui-builder/blob/299542293628aabad6617fc7ff41ae41e1335aaa/src/fragment-components/a-tag.tsx#L157). Based on the comment, it seems to be already there 😆 so the comments can now be removed.

Yes, there should be a event emitted for `onClose`. which should be added. @zvonimirfras Any thoughts on how onClose should be added? I'm thinking something similar to [a-checkbox.tsx](https://github.com/IBM/carbon-ui-builder/blob/main/src/fragment-components/a-checkbox.tsx#L106).


---

Thanks for the feedback & suggestion, much appreciated! Not sure why you add to install CRA & sass globally though... will have someone take a look at this. However, feel free to contribute a fix for setting environment variables in windows!

**Comment by miqh** [comment]
Appreciate the insight @Akshat55.

Ah, I wasn't aware that the Angular implementation targeted an older version of Carbon Design System. I've only just located the implementation-agnostic documentation for it [here](https://v10.carbondesignsystem.com/).

More broadly, can one expect that the React implementation for v10 is complete? (i.e. Adheres to all features explained by the design documentation.) If it is, then I can use this fact to also background check if any Angular component implementations are missing functionality that's otherwise expected.

When I find some time I'll try get around to making some changes to address the consistency of code exports across various components here.

---

I can also double-check what I said about running a fresh setup of this project on a Windows machine. I'm pretty sure the environment variable issue I pointed out still stands as a problem regardless.
